### DISCUSSION TITLE
Allow uploading single photo in mobile application

### DISCUSSION THREAD

**Discussion by aleksvujic** [ente-io/ente#2655] [Enhancements]
I am using Ente 0.9.16 on iOS (iPhone 12). On the home page, there is a button to add photos in the upper right corner:

<img src="https://github.com/user-attachments/assets/52b91ebb-2834-4a6f-94ff-8df7ade199b1" width=40%>

When it is clicked, a "Select folders for backup" dialog shows up which allows the user to backup the whole folder.

<img src="https://github.com/user-attachments/assets/de911f25-d5df-4013-baf4-515ddf5dfdbd" width=40%>

However, what if user doesn't want to upload a whole album to Ente, but just some photos from the album? Why is it not possible to upload just specific photos from the camera roll? You only allow whole album upload.

**Proposal:** Allow users on mobile devices to upload selected photos, not just whole albums.

**Comment by aqsede** [comment]
go to Albums/Gallery there is "On device" at the top, open the folder where that specific picture is, and open that picture, there will be upload button. You can do that one by one.
